nat-lab/tests/test_pq.py

```diff
@@ -130,3 +142,125 @@ async def test_pq_vpn_connection(
+            ),
+            "10.0.254.1",
+        ),
+        pytest.param(
+            SetupParameters(
+                connection_tag=ConnectionTag.DOCKER_CONE_CLIENT_1,
+                adapter_type=AdapterType.LinuxNativeWg,
+                connection_tracker_config=generate_connection_tracker_config(
+                    ConnectionTag.DOCKER_CONE_CLIENT_1,
+                    stun_limits=ConnectionLimits(1, 1),
+                    nlx_1_limits=ConnectionLimits(2, 2),
+                ),
+                is_meshnet=False,
+            ),
+            "10.0.254.1",
+            marks=pytest.mark.linux_native,
+        ),
+        pytest.param(
+            SetupParameters(
+                connection_tag=ConnectionTag.WINDOWS_VM_1,
+                adapter_type=AdapterType.WindowsNativeWg,
+                connection_tracker_config=generate_connection_tracker_config(
+                    ConnectionTag.WINDOWS_VM_1,
+                    stun_limits=ConnectionLimits(1, 1),
+                    nlx_1_limits=ConnectionLimits(2, 2),
+                ),
+                is_meshnet=False,
+            ),
+            "10.0.254.7",
+            marks=[
+                pytest.mark.windows,
+            ],
+        ),
+        pytest.param(
+            SetupParameters(
+                connection_tag=ConnectionTag.WINDOWS_VM_1,
+                adapter_type=AdapterType.WireguardGo,
+                connection_tracker_config=generate_connection_tracker_config(
+                    ConnectionTag.WINDOWS_VM_1,
+                    stun_limits=ConnectionLimits(1, 1),
+                    nlx_1_limits=ConnectionLimits(2, 2),
+                ),
+                is_meshnet=False,
+            ),
+            "10.0.254.7",
+            marks=[
+                pytest.mark.windows,
+            ],
+        ),
+        pytest.param(
+            SetupParameters(
+                connection_tag=ConnectionTag.MAC_VM,
+                adapter_type=AdapterType.BoringTun,
+                connection_tracker_config=generate_connection_tracker_config(
+                    ConnectionTag.MAC_VM,
+                    stun_limits=ConnectionLimits(1, 1),
+                    nlx_1_limits=ConnectionLimits(2, 2),
+                ),
+                is_meshnet=False,
+            ),
+            "10.0.254.7",
+            marks=pytest.mark.mac,
+        ),
+    ],
+)
+async def test_pq_vpn_rekey(
+    alpha_setup_params: SetupParameters,
+    public_ip: str,
+) -> None:
+    # Set rekey interval to some small value
+    alpha_setup_params.features.post_quantum_vpn.rekey_interval_s = 2
+
+    async with AsyncExitStack() as exit_stack:
+        env = await exit_stack.enter_async_context(
+            setup_environment(exit_stack, [alpha_setup_params])
+        )
+
+        client_conn, *_ = [conn.connection for conn in env.connections]
+        client_alpha, *_ = env.clients
+
+        ip = await testing.wait_long(stun.get(client_conn, config.STUN_SERVER))
```

Comment:
new changes appeared in main, could you remove `testing.wait_long` wrapper from `stun.get`, since `timeout` is now parameter of `stun.get`

Reply:
Done